### PR TITLE
Refactor footer helper to accommodate both services

### PR DESCRIPTION
**Pull request by CatalinVoineag**
## Context

This should make the footer helper a bit easier to scale for both services.

## Changes proposed in this pull request

footer_helper.rb

### CODE REVIEW

**Comment by CatalinVoineag** [comment]
[Refactor footer_helper](https://trello.com/c/ffUATsPW/307-refactor-footerhelper)

File: app/helpers/footer_helper.rb [hunk 1]
```diff
@@ -1,11 +1,31 @@
 module FooterHelper
   def footer_meta_items
+    if current_service == :claims
+      claims_footer_meta_items
+    else
+      placements_footer_meta_items
+    end
```
Comment (Nitemaeric) [inline app/helpers/footer_helper.rb:7]: Let's avoid using an `else` statement. If `current_service` is not either `:claims` or `:placements`, we want to immediately be able to see that.

```suggestion
    {
      claims: claims_footer_meta_items,
      placements: placements_footer_meta_items
    }.fetch current_service
```

**Review by Kizr — CHANGES_REQUESTED** [review]
Please could we add a test for `footer_meta_items`? Given this app serves two purposes it would give us more confidence that we haven't mistakenly broken the part we can't see.

**Review by Kizr — APPROVED** [review]
Awesome, thank you!

**Pull request merged**